x-pack/plugins/actions/server/builtin_action_types/servicenow/index.ts

```diff
@@ -27,18 +28,26 @@ import {
   PushToServiceResponse,
   ExecutorSubActionCommonFieldsParams,
   ServiceNowExecutorResultData,
+  ExecutorSubActionGetChoicesParams,
 } from './types';
 
-export type ActionParamsType = TypeOf<typeof ExecutorParamsSchema>;
+export type ActionParamsType =
+  | TypeOf<typeof ExecutorParamsSchemaIM>
+  | TypeOf<typeof ExecutorParamsSchemaSIR>;
 
 interface GetActionTypeParams {
   logger: Logger;
   configurationUtilities: ActionsConfigurationUtilities;
 }
 
-export const ActionTypeId = '.servicenow';
+const serviceNowIMTable = 'incident';
+const serviceNowSIRTable = 'sn_si_incident';
+
+export const ServiceNowIMActionTypeId = '.servicenow';
```

Comment:
I'm not sure if we need to rename existing actionTypeId for ServiceNow to ITSM related (probably migration should be involved for the existing connectors) but we definitely need to change the connector name.